apisix/core/id.lua

```diff
@@ -20,36 +20,38 @@
 -- @module core.id
 
 local fetch_local_conf = require("apisix.core.config_local").local_conf
-local try_read_attr    = require("apisix.core.table").try_read_attr
-local log              = require("apisix.core.log")
-local uuid             = require('resty.jit-uuid')
-local smatch           = string.match
-local open             = io.open
-
-
+local try_read_attr = require("apisix.core.table").try_read_attr
+local profile = require("apisix.core.profile")
+local log = require("apisix.core.log")
+local uuid = require("resty.jit-uuid")
+local smatch = string.match
+local open = io.open
+local lyaml = require("lyaml")
+local type = type
+local ipairs = ipairs
+local string = string
+local math = math
 local prefix = ngx.config.prefix()
 local apisix_uid
+local pairs = pairs
 
-local _M = {version = 0.1}
-
+local _M = { version = 0.1 }
 
 local function rtrim(str)
     return smatch(str, "^(.-)%s*$")
 end
 
-
 local function read_file(path)
     local file = open(path, "rb") -- r read mode and b binary mode
     if not file then
         return nil
     end
 
-    local content = file:read("*a")  -- *a or *all reads the whole file
+    local content = file:read("*a") -- *a or *all reads the whole file
```

Comment:
better no not introduce changes like this, as this is already a big PR. Plus, a PR should only contain changes meaningful to the PR title.